tests/integration/operator-integ-suite-ocp.sh

```diff
@@ -24,14 +24,47 @@ set -u
 # Print commands
 set -x
 
-# deploy operator in OCP
-echo "--------------------------------"
-echo "Check that istio operator is running in OCP"
-echo "--------------------------------"
-"${WD}"/check-operator.sh
-
-# create a Istio Helm Install CP and test httpbin
-echo "--------------------------------"
-echo "Create a IstioHelmInstall Control Plane and test httpbin"
-echo "--------------------------------"
-"${WD}"/control-plane-test.sh
+OPERATOR_NS="${OPERATOR_NS:-istio-operator}"
+OPERATOR_NAME="${OPERATOR_NAME:-istio-operator}"
+CONTROL_PLANE_NS="${CONTROL_PLANE_NS:-default}"
+
+BRANCH="${BRANCH:-maistra-3.0}"
+ISTIO_HELM_INSTALL="https://raw.githubusercontent.com/maistra/istio-operator/${BRANCH}/config/samples/maistra.io_v1_istiohelminstall.yaml"
```

Comment:
Instead of pointing to a URL, just use the file from this repo. This allows the test to actually test current code, not code that's already deployed.

Suggestion: `ISTIO_HELM_INSTALL="${WD}/../../config/samples....."`

Reply:
Change made.